git-hub

```diff
@@ -7,37 +7,67 @@ import sys
 import textwrap
 import json
 import toml
+import app
 import os
+from datetime import datetime
 from urllib.request import urlopen, Request
-
 from os.path import join as pjoin
 
 
 def login():
-    """parses out username and repo"""
+    """Retrieves the user's remote.
+
+    Returns
+    -------
+    username : string
+        username of the remote
+    repo : string
+        repository of the remote
+    remote : string
+        returns remotes
```

Comment:
What does this description mean?

Reply:
repo is supposed to be the repository of the github user. Remotes is supposed to be a string of all of the remotes in the git file(as in when I type in 'git remote -v' it returns the string of all of the info we get from git remote -v. Is that a better description?

Reply:
Maybe call it `remotes` and describe it as `Git remotes: the output of `git remote -v``.

Reply:
My doc string right now looks like: 

    """Retrieves the user's remote.
    For example consider running 'git remote -v' and getting back:
        origin  git@github.com:test_user/test_repo.git (fetch)
        origin git@github.com:test_user/test_repo.git (push)
    
    Returns
    -------
    username : string
        username of the remote, i.e. test_user
    repo : string
        origin repository of the github user, i.e. test_repo
    remotes : string
        Git remotes: the output of 'git remote -v'

    """

Is this good? I'm not sure what the format for the examples should be(based on the comment below on adding examples).